src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -197,11 +199,9 @@ export const Browser: FunctionComponent<BrowserProps> = (
   }, [props.browserQueryParams.focus]);
 
   useEffect(() => {
-    const chrLocationForGenome = props.chrLocation[props.activeGenomeId] || [
-      '',
-      0,
-      0
-    ];
+    const chrLocationForGenome = props.chrLocation[
+      props.activeEnsObjectId[props.activeGenomeId]
+    ] || ['', 0, 0];
```

Comment:
Is there a point in having a default chromosome location ['', 0, 0] from the genome browser perspective, @ens-ds23?